br-ext/package/optee_client/S30optee

```diff
@@ -19,6 +19,10 @@ case "$1" in
 			chown root:ion /dev/ion && chmod 0660 /dev/ion
 			[ $? = 0 ] && echo "OK" || echo "FAIL"
 		fi
+		printf "Create and set permissions on /data... "
+		mkdir -p /data/tee/logs && chown -R tee:tee /data/tee && \
+			chmod 0770 /data/tee
```

Comment:
TBH this looks hackish and duplicates https://github.com/OP-TEE/build/blob/master/br-ext/package/optee_client/optee_client.mk#L33. Moreover, only NFS seems to benefit from it (other platforms thn Rpi3 with NFS certainly work fine already). Perhaps this just hides some other problem? Or we could make a NFS-specific fix instead?
What is currently in `optee_client.mk` seems to be the proper way.

Is it necessary to create `logs`? What happens if it does not exists? I suppose logs are silently discarded. Perhaps `/data/tee/logs` should be added to `optee_client.mk` instead.